roles/yoda_portal/templates/flask.cfg.j2

```diff
@@ -14,3 +18,56 @@ SESSION_COOKIE_SECURE       = True
 PERMANENT_SESSION_LIFETIME  = {{ flask_permanent_session_lifetime }} 
 SESSION_USE_SIGNER          = True
 SESSION_FILE_DIR            = '{{ flask_session_file_dir }}'
+
+# iRODS authentication configuration
+IRODS_ICAT_HOSTNAME = 'combined.yoda.test'
```

Comment:
Can we replace hostname, port and zone with existing variables? (irods_icat_fqdn etc.)